Dockerfile

```diff
@@ -94,6 +94,8 @@ COPY --from=assets-precompile /app /app
 COPY --from=assets-precompile /usr/local/bundle/ /usr/local/bundle/
 COPY --from=middleman /public/ /app/public/
 
+RUN echo "cd /app && /usr/local/bundle/bin/bundle exec rails c" > /root/.ash_history
```

Comment:
```suggestion
RUN echo "cd /app && /usr/local/bundle/bin/bundle exec rails c" > /root/.bash_history
```

Reply:
i'm pretty sure alpine uses `.ash_history`, it worked when i tested it

Reply:
faire enough, I thought this was a typo